src/openrct2/command_line/CommandLine.hpp

```diff
@@ -89,27 +89,12 @@ enum
 
 constexpr char NAC = '\0';
 
-#define ExampleTableEnd                                                                                                        \
-    {                                                                                                                          \
-        nullptr, nullptr                                                                                                       \
-    }
-#define OptionTableEnd                                                                                                         \
-    {                                                                                                                          \
-        UINT8_MAX, nullptr, NAC, nullptr, nullptr                                                                              \
-    }
-#define CommandTableEnd                                                                                                        \
-    {                                                                                                                          \
-        nullptr, nullptr, nullptr, nullptr, nullptr                                                                            \
-    }
+#define ExampleTableEnd { nullptr, nullptr }
+#define OptionTableEnd { UINT8_MAX, nullptr, NAC, nullptr, nullptr }
+#define CommandTableEnd { nullptr, nullptr, nullptr, nullptr, nullptr }
 
-#define DefineCommand(name, params, options, func)                                                                             \
-    {                                                                                                                          \
-        name, params, options, nullptr, func                                                                                   \
-    }
-#define DefineSubCommand(name, subcommandtable)                                                                                \
-    {                                                                                                                          \
-        name, "", nullptr, subcommandtable, nullptr                                                                            \
-    }
+#define DefineCommand(name, params, options, func) { name, params, options, nullptr, func }
```

Comment:
You can use `// NOLINTNEXTLINE(cppcoreguidelines-macro-usage)` if you think this should stay
https://github.com/OpenRCT2/OpenRCT2/blob/1dead6a4c8f934a57001fe228e60fa385390b685/src/openrct2/util/Prefetch.h#L13-L15

```suggestion
// NOLINTNEXTLINE(cppcoreguidelines-macro-usage)
#define DefineCommand(name, params, options, func) { name, params, options, nullptr, func }
```

Reply:
I decided instead to just get rid of this macros.